### PR TITLE
(#4942) - 5.3.0 blog post and config changes

### PR DESCRIPTION
**Pull request by nolanlawson**
Adds the blog post with the changelog and makes the necessary
`tin` updates.

### CODE REVIEW

File: docs/_posts/2016-03-04-pouchdb-5.3.9-sqlite-support-in-node.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+#### Features
+
+* Support WebSQL in Node via node-websql ([#4915](https://github.com/pouchdb/pouchdb/issues/4915))
+
+#### Bugfixes
+
+* Escape unicode in auth header ([#4903](https://github.com/pouchdb/pouchdb/issues/4903))
+* Fix cache-busting in Edge ([#4827](https://github.com/pouchdb/pouchdb/issues/4827))
+* When when re-using a destroyed database ([#4339](https://github.com/pouchdb/pouchdb/issues/4339))
```
Comment (eskatos) [inline docs/_posts/2016-03-04-pouchdb-5.3.9-sqlite-support-in-node.md]: Small type: this should read "When re-using" ..

Comment (nickcolley) [inline docs/_posts/2016-03-04-pouchdb-5.3.9-sqlite-support-in-node.md]: "Better error when trying to re-use a destroyed database"

Comment (nolanlawson) [inline docs/_posts/2016-03-04-pouchdb-5.3.9-sqlite-support-in-node.md]: my bad, early morning :stuck_out_tongue:

**Comment by nickcolley** [comment]
LGTM :+1:

**Comment by nolanlawson** [comment]
Sweet, will publish!

**Pull request merged**